lib/parlour/rbi_generator/attribute.rb

```diff
@@ -47,6 +51,24 @@ def initialize(generator, name, kind, type, &block)
       # @return [Symbol]
       attr_reader :kind
 
+      sig { returns(T::Boolean) }
+      # Whether this attribute belongs to the singleton class.
+      attr_reader :class_attribute
+
+      sig { override.params(other: Object).returns(T::Boolean) }
+      # Returns true if this instance is equal to another attribute.
+      #
+      # @param other [Object] The other instance. If this is not a {Attribute}
+      #   (or a subclass of it), this will always return false.
+      # @return [Boolean]
+      def ==(other)
+        T.must(
```

Comment:
is this a valid use of `T.must`?

Reply:
Seems to be, because `super` is untyped: https://github.com/sorbet/sorbet/issues/1068